schemas/vertices/features.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "description": "A component of a DNA sequence, such as a CDS, mRNA, etc.",
+  "required": [
+    "_key",
+    "type",
+    "location"
+  ],
+  "properties": {
+    "_key": {
+      "type": "string",
+      "title": "Content hash",
+      "description": "Hash of the DNA sequence for this feature."
+    },
+    "protein_translation": {
+      "type": "string",
+      "title": "Protein translation",
+      "description": "Longest coded protein (representative protein for splice variants)"
```

Comment:
Also can be big. Might want to drop this in future

Reply:
We can keep it optional and not import it for now